hangups/conversation.py

```diff
@@ -544,6 +593,29 @@ async def set_notification_level(self, level):
             )
         )
 
+    async def modify_otr_status(self, otr=hangouts_pb2.
+                                OFF_THE_RECORD_STATUS_OFF_THE_RECORD):
```

Comment:
I think the argument should be a `bool` for consistency with this class' existing `is_off_the_record` property.

Reply:
Done -- I've also removed the default as the method name was ambiguous as to what it would do.